discovery-provider/integration_tests/queries/test_get_tips.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+import pytest
+from integration_tests.utils import populate_mock_db
+from src.queries.get_tips import get_tips, GetTipsArgs
+from src.utils.db_session import get_db
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def test_entities():
+    return {
+        "users": [
+            {"user_id": 1, "handle": "user1"},
+            {"user_id": 2, "handle": "user2"},
+            {"user_id": 3, "handle": "DontShowMyTips"},
+        ],
+        "follows": [
+            {"follower_user_id": 1, "followee_user_id": 2},
+            {"follower_user_id": 1, "followee_user_id": 3},
+        ],
+        "user_tips": [
+            {
+                "slot": 0,
+                "signature": "abcdefg",
+                "sender_user_id": 1,
+                "receiver_user_id": 2,
+                "amount": 100000000,
+            },
+            {
+                "slot": 1,
+                "signature": "abcdefg",
+                "sender_user_id": 1,
+                "receiver_user_id": 3,
+                "amount": 100000000,
+            },
+        ],
+        "aggregate_user_tips": [
+            {"sender_user_id": 1, "receiver_user_id": 2, "amount": 100000000},
+            {"sender_user_id": 1, "receiver_user_id": 3, "amount": 100000000},
+        ],
+    }
+
+
+def test_exclude_receivers_from_query(app, test_entities):
+    # Add pagination variables to only retrieve first tip result
+    with app.test_request_context("?limit=1&offset=0"):
```

Comment:
I originally based this test file on another test file which was passing `limit` and `offset` using `data`. However, our [pagination args parser](https://github.com/AudiusProject/audius-protocol/blob/main/discovery-provider/src/queries/query_helpers.py#L1087) reads them from `args`, which is populated from _query params_. The only way I found to emulate this in the docs is to initialize the request context with a URL containing query parameters.

https://flask.palletsprojects.com/en/1.1.x/testing/#other-testing-tricks